aeron-client/src/main/java/io/aeron/logbuffer/LogBufferDescriptor.java

```diff
@@ -135,6 +135,17 @@ public class LogBufferDescriptor
      */
     public static final int LOG_DEFAULT_FRAME_HEADER_OFFSET;
 
+    /**
+     * Offset within the log metadata where the receive buffer size (rcvBuf) is stored.
+     */
+    public static final int LOG_RCVBUF_LENGTH_OFFSET;
+
+    /**
+     * Offset within the log metadata where the send buffer size (sndBuf) is stored.
+     */
+    public static final int LOG_SNDBUF_LENGTH_OFFSET;
```

Comment:
Naming, i.e. should be `LOG_SOCKET_SNDBUF_LENGTH_OFFSET`.

Reply:
Done